### PR TITLE
Bump libocr version

### PR DESCRIPTION
**Pull request by george-dorin**
-Bump libocr version
-Fix libocr interface
-Send ReportTimestamp to EA telemetry

### CODE REVIEW

File: core/services/ocrcommon/data_source.go [hunk 1]
```diff
@@ -85,7 +94,7 @@ func NewInMemoryDataSource(pr pipeline.Runner, jb job.Job, spec pipeline.Spec, l
 	}
 }
 
-var _ ocrtypes.DataSource = (*dataSource)(nil)
+//var _ ocr1types.DataSource = (*dataSource)(nil)
```
Comment (jmank88) [inline core/services/ocrcommon/data_source.go:97]: Remove or restore?

Comment (george-dorin) [reply core/services/ocrcommon/data_source.go:97]: TBH, I don't get why that is there, commented it to see if anything breaks and forgot about it. I guess we can remove it?

Comment (samsondav) [reply core/services/ocrcommon/data_source.go]: It verifies that `*dataSource` conforms to `ocr1types.DataSource`. Keeping it makes that explicit.

Comment (samsondav) [inline core/services/ocrcommon/data_source.go]:
```suggestion
var _ ocr1types.DataSource = (*dataSource)(nil)
```

**Review by samsondav — CHANGES_REQUESTED** [review]

**Review by jmank88 — APPROVED** [review]

File: core/services/ocrcommon/data_source.go (hunk 1 above)
Comment (george-dorin) [reply core/services/ocrcommon/data_source.go]: Restored in [678a530 (#8959)](https://github.com/smartcontractkit/chainlink/pull/8959/commits/678a53083c1fdab70f274c761a44c0956decb8f4)

File: core/services/ocrcommon/data_source.go [hunk 2]
```diff
@@ -33,37 +33,46 @@ type inMemoryDataSource struct {
...
 
-// dataSource uses inMemoryDataSource and implements capturing the result to be stored in the DB
-type dataSource struct {
+type dataSourceBase struct {
 	inMemoryDataSource
 	runResults chan<- pipeline.Run
 }
 
-// dataSourceV2 implements dataSource with the proper Observe return type
+// dataSourceV2 implements dataSourceBase with the proper Observe return type for ocr1
```
Comment (krehermann) [inline core/services/ocrcommon/data_source.go:41]: typo in comment dataSourceV2 -> dataSource

Comment (george-dorin) [reply core/services/ocrcommon/data_source.go]: Fixed in [`7581ff2` (#8959)](https://github.com/smartcontractkit/chainlink/pull/8959/commits/7581ff2e8c10c735c1560553aa79ce3cf1b9def4)

**Review by krehermann — DISMISSED** [review]

**Review by jmank88 — DISMISSED** [review]

**Review by vyzaldysanchez — APPROVED** [review]

**Review by pinebit — APPROVED** [review]

**Pull request merged**